src/toil/wdl/wdltoil.py

```diff
@@ -738,12 +799,23 @@ class ToilWDLStdLibBase(WDL.StdLib.Base):
     """
     Standard library implementation for WDL as run on Toil.
     """
-    def __init__(self, file_store: AbstractFileStore, execution_dir: Optional[str] = None, enforce_existence: bool = True):
+    def __init__(
+        self,
+        file_store: AbstractFileStore,
+        task_path: str,
+        execution_dir: Optional[str] = None,
+        enforce_existence: bool = True,
+        share_files_with: Optional["ToilWDLStdLibBase"] = None
+    ) -> None:
         """
         Set up the standard library.
-
+        
+        :param task_path: Dotted WDL name of the part of the wrokflow this library is working for.
         :param execution_dir: Directory to use as the working directory for workflow code.
-        :param enforce_existence: If true, then if a file is detected as nonexistent, raise an error. Else, let it pass through
+        :param enforce_existence: If true, then if a file is detected as
+            nonexistent, raise an error. Else, let it pass through
+        :param share_files_with: Use the same file upload and download paths as
+            the provided standard library.
```

Comment:
This description makes `share_files_with` sound like a `bool`.  Could it be a little more descriptive?